### DISCUSSION TITLE
Unable to OAuth with Rocket.Chat

### DISCUSSION THREAD

**Discussion by sanyappc** [requarks/wiki#6783] [Error / Bug Report]
We updated Rocket.Chat server to 6.4.1 and now Rocket.Chat OAuth requires an optional parameter 'state'

I created a corresponding [issue](https://github.com/RocketChat/Rocket.Chat/issues/30423) in the Rocket.Chat repository. But it seems easier to add ?state=1 to [authorizationURL](https://github.com/requarks/wiki/blob/8932d15c0c0197aecb7962aada5f01a47e4a2f5f/server/modules/authentication/rocketchat/authentication.js#L38) in OAuth2Strategy

**Comment by KarolKski** [comment]
+1 We're also experiencing this issue since.
